src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -123,9 +130,11 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	}
 
 	ngOnInit() {
+		this.setBookOpenedContextKey();
```

Comment:
I don't believe you're using this context key anymore

Reply:
Ah yes, removed.